### PR TITLE
Update calculateAPY to round to number

### PR DESCRIPTION
**Pull request by Jibz1**

### CODE REVIEW

File: apps/explorer/src/components/validator/calculateAPY.ts [hunk 1]
```diff
@@ -3,6 +3,21 @@
 
 import type { ActiveValidator } from '~/pages/validator/ValidatorDataTypes';
 
+const ROUNDING_MODE = 4;
+
+function roundTo(number: number, decimals: number) {
```
Comment (Jordan-Mysten) [inline apps/explorer/src/components/validator/calculateAPY.ts:8]: Much easier way to do this:
```typescript
const rounded = parseFloat(number.toFixed(4));
```

Comment (Jibz1) [reply apps/explorer/src/components/validator/calculateAPY.ts]: done.

File: apps/explorer/src/components/validator/calculateAPY.ts [hunk 2]
```diff
@@ -3,6 +3,8 @@
 
 import type { ActiveValidator } from '~/pages/validator/ValidatorDataTypes';
 
+const ROUNDING_MODE = 4;
```
Comment (Jordan-Mysten) [inline apps/explorer/src/components/validator/calculateAPY.ts:6]: Not blocking but probably would call this something more useful like `APY_DECIMALS`

File: apps/wallet/src/ui/app/staking/calculateAPY.ts [hunk 3]
```diff
@@ -3,6 +3,8 @@
 
 import { type ActiveValidator } from '@mysten/sui.js';
 
+const ROUNDING_MODE = 4;
```
Comment (Jordan-Mysten) [inline apps/wallet/src/ui/app/staking/calculateAPY.ts:6]: Same here, would maybe tweak naming.

**Review by Jordan-Mysten — APPROVED** [review]

**Pull request merged**